include/srdfdom/model.h

```diff
@@ -195,6 +195,19 @@ class Model
     std::string name_;
   };
 
+  // Some joints may have additional numerical properties.
+  struct JointProperty
+  {
+    /// The name of the joint that this property belongs to
+    std::string joint_name_;
+
+    /// The name of the property
+    std::string property_name_;
+
+    /// The value of the property
+    double value_;
```

Comment:
maybe make `value_` a std::string and leave conversion to desired type at a later point?